Gemfile

```diff
@@ -19,6 +19,7 @@ gem 'rbnacl', '~> 4.0.2'
 gem 'rbnacl-libsodium', '~> 1.0.16'
 gem 'bcrypt_pbkdf', '>= 1.0', '< 2.0'
 gem 'jaro_winkler', '~> 1.5.6'
+gem 'dry-inflector'
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ should this get a version constraint, just as best practice?

Reply:
Definitely !